tests/Aspire.Templates.Tests/NewUpAndBuildSupportProjectTemplatesTests.cs

```diff
@@ -16,6 +16,7 @@ public class NewUpAndBuildSupportProjectTemplates(ITestOutputHelper testOutput)
     [MemberData(nameof(TestDataForNewAndBuildTemplateTests), arguments: ["aspire-xunit", "--xunit-version v2"])]
     [MemberData(nameof(TestDataForNewAndBuildTemplateTests), arguments: ["aspire-xunit", "--xunit-version v3"])]
     [MemberData(nameof(TestDataForNewAndBuildTemplateTests), arguments: ["aspire-xunit", "--xunit-version v3mtp"])]
+    [Trait("category", "basic-build")]
```

Comment:
I was going to ask -- will this b uild the templates each way to get every package ref,  but this test shows yes